catalyst_voices/packages/catalyst_voices_localization/lib/l10n/intl_en.arb

```diff
@@ -819,6 +819,33 @@
   "@keychainDeletedDialogInfo": {
     "description": "An info on keychain deleted dialog"
   },
+  "registrationCompletedTitle": "Catalyst account setup",
+  "registrationCompletedSubtitle": "Completed!",
+  "registrationCompletedSummaryHeader": "Summary",
+  "registrationCompletedKeychainTitle": "Catalyst Keychain created",
+  "registrationCompletedKeychainInfo": "You created a Catalyst Keychain, backed up its seed phrase and set an unlock password.",
+  "registrationCompletedWalletTitle": "Cardano {walletName} wallet selected",
+  "@registrationCompletedWalletTitle": {
+    "placeholders": {
+      "walletName": {
+        "type": "String"
+      }
+    }
+  },
+  "registrationCompletedWalletInfo": "You selected your {walletName} wallet as primary wallet for your voting power.",
+  "@registrationCompletedWalletInfo": {
+    "placeholders": {
+      "walletName": {
+        "type": "String"
+      }
+    }
+  },
+  "registrationCompletedRolesTitle": "Catalyst roles selected",
+  "registrationCompletedRolesInfo": "You linked your Cardano wallet and selected  Catalyst roles via a signed transaction.",
+  "registrationCompletedRole1x": "1x",
```

Comment:
how about we put this as string literal into code? doesn't seem like it needs to be localized